### ISSUE TITLE
Resource URL in message template does not work when processed by cronjob

### ISSUE THREAD

**Issue by veggiematts** [Coral-erm/Coral#46]
When an email is sent using templates, some variables are expanded, like ResourceTitle or ResourceID (see resources/admin/emails/)

The same goes for the resource url variable: ResourceRecordURL

However, this does not work when the message is processed by a cronjob.

Indeed, the function used to expand the URL variable (getCORALURL) relies on $_SERVER variables, which are not populated when in cron environment.

The produced URL looks like this: `http://:/home/coral/www/resources/resource.php?resourceID=` , which is wrong.

Some fix ideas:
 - Store the CORAL URL in configuration.ini
 - use curl or wget in the crontab, like this: 
`/usr/bin/curl http://domain.tld/path/to/cron-script.php &> /dev/null`

**Comment by jeffnm** [comment]
Should we update this documentation: https://github.com/Coral-erm/Coral/wiki/CORAL-Technical-Documentation#resources-alerts-scheduling to reference the wget/curl methods of invoking the sendAlerts.php file rather than the PHP CLI method?

**Comment by veggiematts** [comment]
Yes, I think so. This method is not ideal, but at least it works, before we can come up with a proper fix.

**Comment by jeffnm** [comment]
The page now mentions curl and wget and the example shows wget.  This is still a bug so I'm leaving the issue open.

**Comment by jbr1nk** [comment]
Yes, please do keep it open, because our cron does not respond with the other two options, so for the time being and because it's internal, we are living with the bad link

________________________________
